.github/workflows/build_test.sh

```diff
@@ -9,7 +9,7 @@ success() { echo >&2 -e "\033[32;1m$1\033[0m"; }
 
 ARGS=(
     "--optimization=0"
-    "--optimization=s"
+    "--optimization=s -Dgnu-efi=true -Defi-cflags=-m32 -Defi-libdir=/usr/lib32"
```

Comment:
on debian-based the canonical location would be `/usr/lib/i386-linux-gnu/` but maybe this is a slightly different use case

Reply:
The gnu-efi package lists ia32 stuff in `/usr/lib32`, so this should be correct.